web/src/features/moreCast2/pages/MoreCast2Page.tsx

```diff
@@ -82,8 +83,13 @@ const MoreCast2Page = () => {
   const [modelType, setModelType] = useState<ModelType>(
     (localStorage.getItem(DEFAULT_MODEL_TYPE_KEY) as ModelType) || DEFAULT_MODEL_TYPE
   )
-  const [fromDate, setFromDate] = useState<DateTime>(DateTime.now())
-  const [toDate, setToDate] = useState<DateTime>(DateTime.now().plus({ days: 2 }))
+
+  const startDateTime = DateTime.now().startOf('day')
```

Comment:
This will probably work, but the following logic might be easier to understand.
```  const startDateTime = DateTime.now()
  const endDateTime = startDateTime.plus({ days: 2 })
  const [fromTo, setFromTo] = useState<DateRange>({
    startDate: startDateTime.toJSDate(),
    endDate: endDateTime.toJSDate()
  })